rule-engine/rule-engine-components/src/main/java/org/thingsboard/rule/engine/delay/TbMsgDelayNode.java

```diff
@@ -46,17 +51,28 @@
                 "does not guarantee that message will be processed even if the \"retry failures and timeouts\" processing strategy will be chosen.",
         icon = "pause",
         uiResources = {"static/rulenode/rulenode-core-config.js"},
-        configDirective = "tbActionNodeMsgDelayConfig"
+        configDirective = "tbActionNodeMsgDelayConfig",
+        version = 1
 )
 public class TbMsgDelayNode implements TbNode {
 
     private TbMsgDelayNodeConfiguration config;
     private Map<UUID, TbMsg> pendingMsgs;
+    private final Set<TimeUnit> supportedTimeUnits = EnumSet.of(
+            TimeUnit.MILLISECONDS, TimeUnit.SECONDS, TimeUnit.MINUTES, TimeUnit.HOURS, TimeUnit.DAYS
```

Comment:
Hi @ashvayka, It will be nice to have some limit of delay timeout. I prefer to set 1 day as a maximum limit, and 1 second as a min limit.

Note: this will not be a backward-compatible fix. Current behavior:

![image](https://github.com/thingsboard/thingsboard/assets/16356590/70159c85-3d23-4b37-ae6d-100e49bc094c)

![image](https://github.com/thingsboard/thingsboard/assets/16356590/c8c5ece2-2327-4d84-913c-ba755dd1e0cd)


As you can see max validation doesn't exist. Min validation is strange.

The only thing that we can do backward-compatible is to use 0 seconds as a minimum delay and in that case, don't schedule any tell-self messages, just do nothing and tell success the original messages.

Since this node is deprecated and we don't have a maximum value validation we suggest adding it, but it can break rule nodes for some existing users